asynctools/asyncipc.nim

```diff
@@ -267,7 +266,7 @@ elif defined(windows):
       interlockedOr(cast[ptr int32](data), 1)
 
     if register:
-      let p = getCurrentDispatcher()
+      let p = getGlobalDispatcher()
```

Comment:
From what I can gather the use of `getCurrentDispatcher` was to access the `ioPort` handle? 1.6 (atleast I think 1.6) introduced [privateAccess](https://nim-lang.org/docs/importutils.html#privateAccess%2Ctypedesc) which allows us to get that handle without needing `HackDispatcher`.

If thats correct, should I make a follow up PR to replace `getCurrentDispatcher` with `getGlobalDispatcher`? (or even just do it in this PR)

Reply:
Oh god no. Don't use `privateAccess` to hack your way into the standard library. Instead export the `ioPort` handle or an accessor proc.

Reply:
Found [getIoHandler](https://nim-lang.org/docs/asyncdispatch.html#getIoHandler%2CPDispatcher) so no need in the end.

Does make me wonder what the point of `HackDispatcher` is then?